### PR TITLE
added points animation assertion

### PR DESCRIPTION
**Pull request by MariaMoroz**

### CODE REVIEW

File: tests/testHomePage.spec.ts [hunk 1]
```diff
@@ -36,8 +36,8 @@ test.describe("Verifycation elements on Home Page", () => {
         await pm.homePage.clickWeek1FirstCheckbox();
         
         await expect(pm.homePage.week1FirstCheckbox).toBeChecked();
-        // await page.waitForTimeout(800);
-        // await expect(pm.homePage.circularProgressbar).toHaveScreenshot({maxDiffPixelRatio : 0.1});
+        await page.waitForTimeout(800);
+        await expect(pm.homePage.circularProgressbar).toHaveScreenshot("pointsBar.png");
```
Comment (yuliahar) [inline tests/testHomePage.spec.ts:40]: Please use expect(await pm.homePage.circularProgressbar.screenshot()).toMatchSnapshot(
			"pointsBar.png",
			{ threshold: 0.6 }
		);
		to make screenshot comparison.

File: tests/testHomePage.spec.ts [hunk 2]
```diff
@@ -36,8 +36,8 @@ test.describe("Verifycation elements on Home Page", () => {
         await pm.homePage.clickWeek1FirstCheckbox();
         
         await expect(pm.homePage.week1FirstCheckbox).toBeChecked();
-        // await page.waitForTimeout(800);
-        // await expect(pm.homePage.circularProgressbar).toHaveScreenshot({maxDiffPixelRatio : 0.1});
+        await page.waitForTimeout(800);
```
Comment (yuliahar) [inline tests/testHomePage.spec.ts:39]: add const ANIMATION_TIME = 800 and use it here

**Review by yuliahar — CHANGES_REQUESTED** [review]

File: tests/testHomePage.spec.ts (hunk 1 above)
Comment (yuliahar) [reply tests/testHomePage.spec.ts:40]: Hm, playwright documentation says: use .toHaveScreenshot for comparing screenshots. Could you please check that test will failed if images are different?

Comment (MariaMoroz) [reply tests/testHomePage.spec.ts:40]: Yes, I rechecked many times and method toHaveScreenshot() works correctly

File: tests/testHomePage.spec.ts (hunk 2 above)
Comment (MariaMoroz) [reply tests/testHomePage.spec.ts:39]: done

**Pull request merged**